### DISCUSSION TITLE
Pros and cons of virtual hackathons

### DISCUSSION THREAD

**Discussion by mishmanners** [github/hackathons#3] [General]
What do you think are the pros and cons of virtual hackathons Vs. s. IRL hacks?

Is there something you can share to suggest good examples of each? What about hybrid events?

Would love to hear your thoughts.

**Comment by HUSK1E** [comment]
Virtual hackathons: ANYONE can join.
IRL hackathons: I like interacting with people. There are people I hang with over coffee ☕ 🍰 🍩 or randomly run into on my way to another room. Those encounters have made me friends, valuable contacts, and I've learned a lot from them. It would have never happened (or been very unlikely to happen in a virtual environment).
I also love just chatting to others, talking about the presentation we just listen to, venting hackathon frustrations and more.

**Reply by EsportsQueen** [reply]
Same here. I feel this too. I do like the fact however with virtual events I can "meet" people from all over the world. It's not exactly the same but I have met a TONNE of really cool people through virtual events during the covid pandemic.
